### PR TITLE
docs: Enable direct links to class/interface methods for `@link`

### PR DESCRIPTION
**Pull request by JeanMeche**
This will address some of the links warnings we were logging.

### CODE REVIEW

File: adev/shared-docs/pipeline/api-gen/rendering/symbol-context.ts [hunk 1]
```diff
@@ -5,14 +5,33 @@
...
+
+export function setCurrentSymbol(symbol: string): void {
+  currentSymbol = symbol;
+}
+
+export function getCurrentSymbol(): string | undefined {
+  return currentSymbol;
+}
+
+export function logWarning(link: string, symbol: string): void {
```
Comment (josephperrott) [inline adev/shared-docs/pipeline/api-gen/rendering/symbol-context.ts:33]: Can we have the name of the function be a little more specific to what it is logging for. Can we have it named something like `logSymbolWarning`, or `logUnknownSymbol`

**Review by josephperrott — APPROVED** [review]
LGTM

Just one comment I would like to have you take a look at.

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by thePunderWoman** [comment]
This PR was merged into the repository by commit dbdd7875dd2fa72c65c86af8a4ce661c8ccec82e.

The changes were merged into the following branches: main, 18.2.x

**Pull request closed without merging**